Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use Python base image with ARG version
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION}
+
+# Set environment variables
+ENV PYTHONUNBUFFERED=1
+
+# Set working directory in the container
+WORKDIR /app
+
+# Copy requirements file and install dependencies
+COPY requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
It would be good to upgrade `pip` before the installation of packages

Reply:
Done!